docfx/docs/getting-started.md

```diff
@@ -121,15 +121,36 @@ To keep CsWin32 generating the referred types you need, add them explicitly to `
 
 ### Support for trimming, AOT, and/or disabling the runtime marshaler
 
-Newer .NET runtime versions may fail for CsWin32 generated code when the application project builds with one or both of these properties set:
+**NEW!** CsWin32 now supports AOT by generating code from an MSBuild task rather than a source generator.
```

Comment:
nit: it supported AOT before. But the support is _better_ now. But this has historically been a sore point of contention so I'd rather not 'officially' document that there was not AOT support before. Can we reword this?

```suggestion
**NEW!** CsWin32 now has improved support for NativeAOT by generating code from an MSBuild task rather than a source generator so that it can leverage other source generators that support `[LibraryImport]` and others.
```